repository-data/application/src/main/resources/hcm-config/configuration/queries/templates/new-banner-folder.yaml

```diff
@@ -0,0 +1,30 @@
+---
+definitions:
+  config:
+    /hippo:configuration/hippo:queries/hippo:templates/new-banner-folder:
```

Comment:
Where is this used?

Reply:
HI @jacsamell, I've installed the banner modules that's bringing all these facilities